assets/js/src/announcements/free_plan_announcement.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import MailPoet from 'mailpoet';
+import addReferralId from 'referral_url_decorator.jsx';
```

Comment:
As discussed, this could be on the `MailPoet` object. Also, the MailPoet URLs could be created by a simple factory, maybe.